ForecastScreen.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { View, Text, StyleSheet, Image, ScrollView, Dimensions } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import { useFonts, Inter_400Regular } from '@expo-google-fonts/inter';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome'; // Import FontAwesome icon
+
+const Tab = createBottomTabNavigator();
+
+const ForecastScreen = ({ data }) => {
+  const [fontsLoaded] = useFonts({
+    Inter_400Regular,
+  });
+
+  if (!data || !fontsLoaded) {
+    return (
+      <View style={styles.container}>
+        <Text>Loading forecast data...</Text>
+      </View>
+    );
+  }
+
+  const forecastDays = data.forecast.forecastday;
+
+  return (
+    <Tab.Navigator
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Tab.Screen
+        name="3 Day Forecast"
+        component={ForecastTab}
+        initialParams={{ data: forecastDays.slice(0, 3) }}
```

Comment:
Same comment about moving navigation logic outside of your screen. If we passed the route param with the # of days to the screen, we wouldn't have to do a slice and we can fetch that # of days for each tab when it's tapped. One of the Tabs would look like this:

```
   <Tab.Screen
        name="3 Day Forecast"
        component={ForecastTab}
        initialParams={{ days: 3 }}
        options={{
          tabBarBadge: 3,
          tabBarIcon: ({ color, size }) => (
            <FontAwesomeIcon name="sun-o" color={color} size={size} /> // Use FontAwesome sunny icon
          ),
        }}
      />
```